roles/istio/defaults/main.yaml

```diff
@@ -2,3 +2,4 @@
 istio_version: 1.9.0
 istio_verify_install: true
 istio_create_seldon_gateway: true
+istio_secure: false
```

Comment:
istio_use_ssl? :) Not a strong opinion but I think we were using use_ssl in other places.